cpp/scripts/run-clang-format.py

```diff
@@ -29,10 +29,10 @@
 DEFAULT_DIRS = [
     "cpp/benchmarks",
     "cpp/include",
-    "cpp/include/cudf",
```

Comment:
did you intend to remove `cpp/include/cudf` here? I didn't see that called out anywhere.

Reply:
If I'm not mistaken, I believe this directory should still be getting checked because parent directory `cpp/include` is listed above - I made this change (along with removing `cpp/include/nvtext`) out of clarity since listing the two subdirectories seemed redundant.

Reply:
That being said, I did forget to call that out specifically in the PR body - if you're okay with this change, I'll go ahead and add it there.

Reply:
My only concern is that we're still doing what we expect. I don't have an issue if this is recursive as it is redundant.